src/OpenRiaServices.Tools/Test/TestHelper.cs

```diff
@@ -95,6 +95,7 @@ public static void ValidateFilesEqual(string relativeTestDir, string relativeDep
 
                 // Generate CMD strings to diff and to copy
                 string tfDiffCommand = "tf diff \"" + referenceFileName + "\" \"" + generatedFileName + "\"\r\n";
```

Comment:
feel free to remove any mentions of "tf" commands (including edit) since only git is relevant